media-source/mediasource-play-hzhong.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>

```

Comment:
Change the name of this file to reflect the test case (e.g. mediasource-sourcenode.html)